### PR TITLE
show-message-when-folder-empty

### LINKED ISSUE #167: Show empty state when a folder is empty
image

If there are no apps in a folder, show the message "This folder is empty".

### PR DESCRIPTION
**Pull request by apoorv1316**
Fixes #167 

- Feature: Show a message if a folder is empty
Screenshot 2021-06-08 at 12 55 43 PM

### CODE REVIEW

**Comment by apoorv1316** [comment]
@Navaneeth-pk Please have a look

File: frontend/src/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -194,6 +194,12 @@ class HomePage extends React.Component {
                             }
                             </>)
                           }
+                          {currentFolder.count == 0  && (
+                            <div>
+                              <img className = "mx-auto d-block" src ="assets/images/icons/empty-folder-svgrepo-com.svg" height="120px"/>
+                              <h3 className= "text-center">This folder is empty</h3>
```
Comment (Navaneeth-pk) [inline frontend/src/HomePage/HomePage.jsx:200]: Can we use `span` tag with `text-body` as the class? Just to be consistent with other pages.

Comment (apoorv1316) [reply frontend/src/HomePage/HomePage.jsx:200]: Sure, will do that

File: frontend/assets/images/icons/empty-folder-svgrepo-com.svg [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+	 viewBox="0 0 56.724 56.724" style="enable-background:new 0 0 56.724 56.724;" xml:space="preserve">
+<g>
+	<path style="fill:#010002;" d="M7.487,46.862c1.111,1.467,2.865,2.275,4.938,2.275h30.598c3.984,0,7.934-3.009,8.991-6.849
```
Comment (Navaneeth-pk) [inline frontend/assets/images/icons/empty-folder-svgrepo-com.svg:6]: Can we set the fill colour to `#434b53` ? Most of the other icons are using this color.

Comment (apoorv1316) [reply frontend/assets/images/icons/empty-folder-svgrepo-com.svg]: @Navaneeth-pk Desired changes done

**Pull request merged**